xbmc/cores/RetroPlayer/buffers/BaseRenderBufferPool.cpp

```diff
@@ -73,16 +73,20 @@ bool CBaseRenderBufferPool::Configure(AVPixelFormat format, unsigned int width,
 
 IRenderBuffer *CBaseRenderBufferPool::GetBuffer(size_t size)
 {
-  if (!m_bConfigured)
+  if (!m_bConfigured) {
+    CLog::Log(LOGDEBUG, "RetroPlayer[RENDER]: buffer pool is not configured");
     return nullptr;
+  }
 
   // If frame size is unknown, set it now
   if (m_frameSize == 0)
     m_frameSize = size;
 
   // Changing sizes is not implemented
-  if (m_frameSize != size)
+  if (m_frameSize != size) {
```

Comment:
Coding style is { on new line. This has the added benefit of lowering the number of deleted lines in a pull request (an important measure)